src/App.js

```diff
@@ -6,11 +6,12 @@ import Details from "./Details";
 
 
 class App extends Component {
+
+
   render() {
     return (
       <Router>
         <div className="App">
-          
           <Route exact path="/" component={Gallery} />
           <Route exact path="/:details" component={Details} />
```

Comment:
I personally would change the ":details" for something more like ":movieDetails", or something more descriptive.
But for the rest is just perfect! 👍 💯 🥇

Reply:
thanks, it makes more sense!!